modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListDisplayContext.java

```diff
@@ -158,6 +171,29 @@ public Map<String, Object> getContext() {
 			).setWindowState(
 				LiferayWindowState.POP_UP
 			).buildString()
+		).put(
+			"cpConfigurationListId",
+			() -> {
+				CPConfigurationList cpConfigurationList =
+					getCPConfigurationList();
+
+				if (cpConfigurationList != null) {
+					return cpConfigurationList.getCPConfigurationListId();
+				}
+
+				return null;
+			}
+		).put(
```

Comment:
`.put` here the data set id you're using for the data set tag in your JSP.

In your case it should be `CPConfigurationFDSNames.PRODUCT_CONFIGURATIONS` I think.